CONTRIBUTING.md

```diff
@@ -0,0 +1,212 @@
+
+**Note** - There is an existing [known issue](https://github.com/facebook/react-native/issues/30277) with the Android SDK that is caused by an assumption in the core React Native Code.
+
+#### iOS 
+
+The native iOS SDK is exposed to React Native Javascript via the use of Objective-C in [RNLinksdk.m](https://github.com/plaid/react-native-plaid-link-sdk/blob/master/ios/RNLinksdk.m). This file exports LinkKit native methods using [RCT_EXPORT_METHOD](https://reactnative.dev/docs/native-modules-ios#export-a-native-method-to-javascript) so they can be called directly in Javascript.
+
+## Development workflow
+
+### Running the example app
+
+You can find information on running the example app in the [README](https://github.com/plaid/react-native-plaid-link-sdk/blob/master/example/README.md).
+
+### iOS changes
+
+To edit the Objective-C files, open `example/ios/plaidRNDemo.xcworkspace` in XCode and find the source files at `Pods > Development Pods > react-native-plaid-link-sdk`.
+
+### Android changes
+
+To edit the Kotlin files, open `example/android` in Android studio and find the source files at `react-native-plaid-link-sdk` under `PlaidRNDemo`.
+
+
+### Typescript changes
+
+Use your editor of choice for editing the Typescript files at the root of the project or in `example/`.
+
+Run `tsc` from the project's root (`npm install --global typescript` if command not found) to compile the typescript source. Ensure the `dist` directory has been created.
+
+Make sure your code passes TypeScript and ESLint. Run the following to verify:
+
+```sh
+npm run lint
+```
+
+To fix formatting errors, run the following:
+
+```sh
+npm run lint --fix
+```
+
+### Testing local changes
+
+
+To install local/private packages across local environment we recommend using [yalc](https://github.com/wclr/yalc).
+
+- Run `yalc publish` in `react-native-plaid-link-sdk` package to publish all the files that should be published in remote NPM registry.
+- Run `yalc add react-native-plaid-link-sdk` in your dependent project, which will copy the current version from the store to your project's `.yalc` folder and inject a `file:.yalc/react-native-plaid-link-sdk` into `package.json`.
+- In your dependent project run `npm install` and `cd ios && bundle install && bundle exec pod install`.
+
+
+### Sending a pull request
+
+> **Working on your first pull request?** You can learn how from this _free_ series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+When you're sending a pull request:
+
+- Prefer small pull requests focused on one change.
+- Review the documentation to make sure it looks good.
+- Follow the pull request template when opening a pull request.
+
+## Code of Conduct
```

Comment:
I'd prefer not to have a CoC in this repo -- happy to chat about it offline if you want, but the tl;dr is that I don't think that our use case for these repos really is relevant for CoCs.  None of our existing repos currently have CoCs.

Reply:
Agreed. I think it was a bit of overkill. I removed the COC.